node-notes-cli/app.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const notebook = require('./data.json');
+
+const command = process.argv[2];
+const ID = notebook.nextId;
+const notes = notebook.notes;
+
+function updateNotebook() {
+  fs.writeFile('data.json', JSON.stringify(notebook, null, 2), err => {
+    if (err) throw err;
+  });
+}
```

Comment:
While this works, it isn't a good way to go. The `updateNotebook` should have a parameter of `noteBook`. As it is now you are using the global variable `noteBook` which isn't safe. The data to be saved should be passed into the function.

Reply:
I refactored my code to fix this, but I don't really understand why it isn't safe. 
Is it because since I am using the global variable notebook, and the function can now only be used with that variable? But by creating a parameter, if I were to have two "notebooks" I could use the same function for both?